bot.ml

```diff
@@ -786,15 +786,158 @@ let callback _conn req body =
+                              ~id:pr.id
+                          else
+                            GitHub_queries.get_team_membership ~bot_info
+                              ~org:"coq" ~team:"pushers"
+                              ~user:comment_info.author
+                            >>= function
+                            | Ok _ -> (
+                                GitHub_mutations.merge_pull_request ~bot_info
+                                  ~pr_id:pr.id
+                                  ~commit_headline:
+                                    (f "Merge PR #%d: %s" pr.issue.number
+                                       comment_info.issue.title)
+                                  ~commit_body:
+                                    ( List.fold_left
+                                        reviews_info.approved_reviews
+                                        ~init:"Reviewed-by:\n" ~f:(fun s r ->
+                                          s ^ f "- %s\n" r)
+                                    ^
+                                    if
+                                      not
+                                        (List.is_empty
+                                           reviews_info.comment_reviews)
+                                    then
+                                      List.fold_left
+                                        reviews_info.comment_reviews
+                                        ~init:"Ack-by:\n" ~f:(fun s r ->
+                                          s ^ f "- %s\n" r)
+                                    else "" )
+                                  ~merge_method:`MERGE
+                                >>= fun () ->
+                                match
+                                  List.fold_left ~init:[] reviews_info.files
+                                    ~f:(fun acc f ->
+                                      if
+                                        string_match
+                                          ~regexp:
+                                            "dev/ci/user-overlays/\\(.*\\)" f
+                                      then Str.matched_group 1 f :: acc
+                                      else acc)
+                                with
+                                | [] ->
+                                    Lwt.return ()
+                                | overlays ->
+                                    GitHub_mutations.post_comment ~bot_info
+                                      ~message:
+                                        (f
+                                           "@%s: Please take care of the \
+                                            following overlays:\n\
+                                            %s"
+                                           comment_info.author
+                                           (List.fold_left overlays ~init:""
+                                              ~f:(fun s o -> s ^ f "- %s\n" o)))
```

Comment:
Instead of putting the list of overlay files, would it be possible to post their content instead (within ```` ``` ``` ````)? This is more relevant information as usually there will be a single overlay file in a given PR but it may define overlays for several external projects.

Reply:
It's possible, we just have to make an additional request for each overlay file to retrieve its content.

Reply:
That sounds worthwhile (in practice, there should only be a single overlay file, so we could limit ourselves to retrieving the first one and signaling if there are others) but let's keep this for a later PR.